packages/expo/build/environment/logging.fx.js

```diff
@@ -1,16 +1,19 @@
 import Constants from 'expo-constants';
-import * as Logs from '../logs/Logs';
-import RemoteLogging from '../logs/RemoteLogging';
-if (Constants.__unsafeNoWarnManifest?.logUrl) {
-    // Enable logging to the Expo dev tools only if this JS is not running in a web browser (ex: the
-    // remote debugger). In Expo Web we don't show console logs in the CLI, so there's no special case needed.
-    if (!isRunningInWebBrowser()) {
-        Logs.enableExpoCliLogging();
-    }
-    else {
-        RemoteLogging.enqueueRemoteLogAsync('info', {}, [
-            'You are now debugging remotely; check your browser console for your application logs.',
-        ]);
+// Metro and terser don't seem to be capable of shaking the imports unless they're wrapped in __DEV__.
+if (__DEV__) {
```

Comment:
@EvanBacon How are you guys using `__DEV__` to treeshake Metro? Even using the `require()` API with __DEV__ still doesn't treeshake Metro in our project with `metro-babel-preset`.